flows-advanced/merge_dependabot_pull_requests.py

```diff
@@ -0,0 +1,173 @@
+"""
+Merge multiple dependabot pull requests across multiple repositories:
+1. Queries ten pull requests (PR) in the repository
+2. For each PR, check if the title matches with the provided title
+3. Automatically approve the PR if there's a matching PR
+4. Leave "@dependabot merge" which will merge the PR if all tests pass
+To create a deployment:
+```
+prefect deployment build merge_dependabot_pull_requests.py:merge_dependabot_pull_requests --name for_prefect_collections
+prefect deployment apply merge_dependabot_pull_requests-deployment.yaml
+```
+
+Pre-requisites:
+`pip install prefect prefect-github`
+"""  # noqa
+
+
+import base64
+from datetime import timedelta
+from pathlib import Path
+from typing import Any, Dict, List
+
+import httpx
+import yaml
+from prefect import flow, get_run_logger, task
+from prefect.tasks import task_input_hash
+
+from prefect_github import GitHubCredentials
+from prefect_github.mutations import add_comment_subject, add_pull_request_review
+from prefect_github.repository import (
+    query_repository_pull_request,
+    query_repository_pull_requests,
+)
+
+BLOCK_NAME = "merge-dependabot-token"
+REPOSITORY_OWNER = "PrefectHQ"
+COLLECTION_AUTHOR = "Prefect"
+DEPENDABOT_PULL_REQUEST_TITLE = "Bump actions/add-to-project from 0.3.0 to 0.4.0"
+
+
+@task(
+    cache_key_fn=task_input_hash,
+    cache_expiration=timedelta(days=1),
+    retries=3,
+    retry_delay_seconds=10,
+)
+def get_collection_names(
```

Comment:
Would this be clearer with the name `get_repository_names`? And then referenced in the flow as `repository_name` instead of `collection_name`?